core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -282,7 +282,7 @@ public int insert(String sql, Object... args) {
      * @return the number of updated inserted
      */
     public int update(String sql, Object... args) {
-        Update stmt = createStatement(sql);
+        Update stmt = createUpdate(sql);
```

Comment:
Yeah, the missing upstream commits I pulled in appear to be rolled in here. Sorry! Still getting used to GitHub's PR system.